SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,38 @@
+header:
+  schema-version: '1.0.0'
+  expiration-date: '2025-01-25T01:00:00.000Z'
+  last-updated: '2024-01-25'
+  last-reviewed: '2024-01-25'
+  project-url: https://github.com/cilium/cilium
+  license: https://github.com/cilium/cilium/blob/main/LICENSE
+project-lifecycle:
+  status: active
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/cilium/cilium/blob/main/MAINTAINERS.md
+  roadmap: https://docs.cilium.io/en/latest/community/roadmap
```

Generally the `/latest` docs are in-development so we try to avoid publicly linking them. `/stable` corresponds to docs for the latest stable release.
```suggestion
  roadmap: https://docs.cilium.io/en/stable/community/roadmap
```